free-programming-books-pt_BR.md

```diff
@@ -304,6 +304,7 @@
 #### Django
 
 * [Tutorial Django Girls](https://tutorial.djangogirls.org/pt/)
+* [Documentação do Django (version 3.1)](https://docs.djangoproject.com/pt-br/3.1/)
```

Comment:
Instead of linking directly to version 3.1, it might be better to ignore the version from the name and URL so that it automatically links to the latest version. If users need a specific version they can navigate to it themselves manually.

```suggestion
* [Documentação do Django](https://docs.djangoproject.com/pt-br/)
```